### DISCUSSION TITLE
What is trick to get it working?

### DISCUSSION THREAD

**Discussion by yabdab** [spatie/ray#681] [General]
Since day one of using the my Ray app, it has been hit or miss if it works or not.
Now I am on brand new M1 Mac , with fresh Valet setup and still same old issue. None of my ray() calls are getting passed to the Ray app.

My question , is there. a way to debug Ray to see if it is working?

My local server settings are set to 127.0.0.1:22 using my username and password.

**Comment by nerdylist** [comment]
I'm having the same issue. Hopefully someone can address this.

**Comment by yabdab** [comment]
One of the reasons I always pass on lifetime license deal, just not reliable. 🤷‍♂️

**Comment by freekmurze** [comment]
For most, it just seems to work. For those that are having problems, do you experience the issue on a vanilla Laravel project (that doesn't run in Docker)?

**Reply by yabdab** [reply]
It is normally in old Codeigniter and non-Laravel  vanilla projects. Sometimes it works, sometime it does not. 🤷‍♂️

**Reply by freekmurze** [reply]
Could you create a project that recreates the issue?

**Reply by yabdab** [reply]
Maybe, but would be easier to debug on my end. I have even installed it globally and still no action. 
I have it set up on a new Mac Studio and it has never worked reliably since day one. 
Just wondering if I am doing something wrong?

Is there a way to debug Ray ( the debugging app ) ?

**Reply by yabdab** [reply]
I just discovered that if I do not enable the Server, then the Ray app works.

Enabled with bolt icon = no work
Disabled with plugin icon = works!

Is this how it is supposed to work?

**Reply by yabdab** [reply]
Looks like Servers panel is only for "remote" servers, not your local machine/server.

I deleted the localhost server and Ray continued to log my local environment entries. 🎉
